### DISCUSSION TITLE
Extended Utility and Sample Projects

### DISCUSSION THREAD

**Discussion by BradKML** [cinemagoer/cinemagoer#341] [Ideas]
Animation Family Tree of Creators (Directed Network)
Original Idea from https://github.com/polakowo/oscarobber

### Tactic 1: Big Picture
1. Gather all animations (search by company and year)
2. Gather cast & crew metadata
3. Create Directed Graph (NetworkX) where the Cast/Crew of Film A points to Creator of Later Film B

### Tactic 2: Small Picture
1. Find one piece of animation and gather its cast & crew
2. Breadth-First Search each cast & crew's work proceeding the original work
3. Rinse and repeat until a tree graph is formed (using NetworkX)

![](https://archive.fo/RiSxC/7350f685476d478557ff9c252b4c8a4e3e1e7608.jpg)
![](https://archive.fo/CYzZa/f318d361f5749981f26b2bf3200b33bbaeeefe38.jpg)

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
